app/src/main/res/values/strings.xml

```diff
@@ -11,7 +11,8 @@
     <string name="monsters_list_tab_small">Small</string>
     <string name="monsters_list_tab_large">Large</string>
     <string name="monsters_detail_tab_summary">Summary</string>
-    <string name="monsters_detail_tab_rewards">Rewards</string>
+    <string name="monsters_detail_tab_rewards_high_rank">High Rank Rewards</string>
```

Comment:
Change these strings to High-Rank and Low-Rank as they're currently too long to fit in a tab title.